sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/Lease.java

```diff
@@ -42,7 +42,11 @@ public interface Lease {
      */
     String getTimestamp();
 
-    ChangeFeedState getContinuationState(
+    ChangeFeedState getIncrementalContinuationState(
```

Comment:
please rename getContinuationStateWireFormatV0 vs. getContinuationStateWireFormatV1 or similar?

Reply:
This one actually has nothing to do with change feed wire format. One is for getting the continuation state for incremental mode, and one for full fidelity mode. 

I guess once I make the lease changes as well, then there will be a good separation of these continuation states based on different lease structure. I might need to create a new lease altogether, will update it then if that's okay with you.

Reply:
what is the difference of the continuationToken between fullFidelity vs incremental?